Documentation/ApiOverview/Fluid/Index.rst

```diff
@@ -94,10 +94,33 @@ In your extension, the following directory structure should be used for Fluid fi
         ├── Partials
         └── Templates
 
-How you structure the files within the respective `Layouts`, `Partials` and `Templates`
-is up to you. If you are using Extbase controller actions in combination with Fluid,
-Extbase clearly defines how files and directories should be named within these directories,
-see :ref:`extbasebook:template-creation-by-example`.
+This directory structure is the convention used by TYPO3 CMS. When using Fluid outside of
+TYPO3 CMS you can use any folder structure you like.
+
+If you are using Extbase controller actions in combination with Fluid,
+Extbase defines how files and directories should be named within these directories.
+Extbase uses the sub-folders of the "Templates" directory to group templates by controller name
```

Comment:
Extbase uses sub directories located within the "Templates"

Reply:
Suggestions look good. I will push another change.